inlang/source-code/website/src/pages/documentation/index.page.tsx

```diff
@@ -143,32 +139,26 @@ export function Page(props: PageProps) {
 						</sl-details>
 					</nav>
 					<Show when={props.markdown} fallback={<p class="text-danger">{props.markdown?.error}</p>}>
-						{/* 
-            rendering on the website is broken due to relative paths and 
-            the escaping of html. it is better to show the RFC's on the website
-            and refer to github for the rendered version than to not show them at all. 
-          */}
-						<div class="w-full justify-self-center mb-8 md:px-6 md:col-span-3">
-							<Show when={currentPageContext.urlParsed.pathname.includes("rfc")}>
-								{/* <Callout variant="warning">
-									<p>
-										The rendering of RFCs on the website might be broken.{" "}
-										<a href="https://github.com/inlang/inlang/tree/main/rfcs" target="_blank">
-											Read the RFC on GitHub instead.
-										</a>
-									</p>
-								</Callout> */}
-							</Show>
-							<div
-								// change the col-span to 2 if a right side nav bar should be rendered
-								class="w-full justify-self-center md:col-span-3"
-							>
+						<div class="flex flex-1 mb-8 md:ml-4 lg:ml-12 min-h-screen">
+							<div class="flex-1 md:max-w-[500px] lg:max-w-[724px] w-full justify-self-center md:col-span-3">
 								<Markdown markdown={props.markdown} />
 								<EditButton href={editLink()} />
 								<Feedback />
 							</div>
 						</div>
 					</Show>
+					<div class="top-[116px] hidden sticky xl:block h-[calc(100%_-_112px)] w-[230px] py-[60px] pl-4">
```

Comment:
I'm missing a overflow in y direction with the custom overflow scrollbar here in case we have too many anchors.

Like already introduced at the left navbar:
```tsx
<nav class="max-h-[96vh] overflow-y-scroll overflow-scrollbar -ml-4 pl-4">
```

Reply:
fixed ✅